debezium-core/src/main/java/io/debezium/jdbc/JdbcConfiguration.java

```diff
@@ -32,6 +32,12 @@
      */
     public static final Field DATABASE = Field.create("dbname",
             "Name of the database");
+    /**
+     * A field for the names of a comma separated list of databases. This field has no default value.
+     * The dbname field should be set to the default database to use from this list.
+     */
+    public static final Field DATABASES = Field.create("dbnames",
```

Comment:
Is it that everything after the `database.` prefix automatically gets assigned to the JDBC configuration? Technically, "dbname" is a JDBC parameter because it can be used to configure the connection but a list of databases isn't really. Do `database.history.*` parameters also get under the JDBC config because of the prefix? If not, we can use the same mechanism and exclude it from here.

Reply:
I moved this field out of JdbcConfiguration and into the SqlServerConfiguration class which is more appropriate.